### DISCUSSION TITLE
Developer Meeting (Nov 1 2023): Upcoming plans Simulation

### DISCUSSION THREAD

**Discussion by knmcguire** [bitcraze/discussions#992] [Announcements]
The next developer meeting will be on the 1th of November, where we will talk about the upcoming plans to improve the Crazyflie simulator. It will be based on the content of the [latest blogpost](https://www.bitcraze.io/2023/10/development-plans-for-crazyflie-simulation/)

Date: Wednesday, Nov 1th, 2023
Time: 3 PM Central European Time (check your [timezone](https://www.worldtimebuddy.com/?qm=1&lid=2673730,5368361,1850147&h=2673730&date=2023-11-1&sln=15-16&hf=0))
Topic: Upcoming plans for simulation and the Crazyflie 
Hosts: @knmcguire
Meeting link: Meeting is over, recording can be found here https://youtu.be/k6x2lbcUwn4 

Agenda
15:00 - 15:30: Presentation, hands-on and discussion
15:30 - 16:00: Generic support questions

Indicate with 👍 if you are planning to join!

**Comment by mwlock** [comment]
Could I, as a general user interested in the state of the simulator, also attend? 😇

**Reply by knmcguire** [reply]
Ofcourse! open to anyone :D I think most attendees are not perse very developer anyway, but it is also to get some feedback on our ideas for the plans

**Reply by krichardsson** [reply]
Of course, everyone is welcome!
As general information, we also always have a section for general questions where we can discuss any topic that is not related to the main theme of the Developer meeting.

**Comment by knmcguire** [comment]
The last blogpost contains some more information too https://www.bitcraze.io/2023/10/development-plans-for-crazyflie-simulation/

**Comment by knmcguire** [comment]
The recording can be found here: https://youtu.be/k6x2lbcUwn4

I also have the slides with links: 

[dev meeting simulation 20231101 (1).pdf](https://github.com/bitcraze/discussions/files/13236218/dev.meeting.simulation.20231101.1.pdf)
